packages/orchestration/src/exos/ica-account-kit.js

```diff
@@ -174,13 +189,34 @@ export const prepareIcaAccountKit = (zone, { watch, asVow }) =>
           });
         },
         /**
+         * This handler fires any time the connection (channel) closes. This
+         * could be due to external factors (e.g. a packet timeout), or a holder
+         * initiated action (`.deactivate()`).
+         *
+         * Here, we clear the connection and addresses from state as they will
+         * change - a new channel will be established if the connection is
+         * reopened.
```

Comment:
consider "if *a* connection is *opened again*" to convey the newness of the connection